ijPlugin/src/main/resources/org/jetbrains/research/tasktracker/ui/main/panel/template/final.html

```diff
@@ -0,0 +1,5 @@
+<div class="container">
+  <p class="big-font">Thank you for taking part in our emotional journey!</p>
```

Comment:
I am not sure about emotional journey here... should we change it to something neutral?